platform/features/hyperlink/res/templates/hyperlink.html

```diff
@@ -0,0 +1,20 @@
+<div class="s-hyperlink" ng-controller="HyperlinkController as hyperlink">
+      <div ng-if="hyperlink.isButton()">
+          <div ng-if="hyperlink.openNewTab()">
+              <a class="s-button"
+              href ={{domainObject.getModel().url}} target="_blank">{{domainObject.getModel().displayText}}</a>
+          </div>
+          <div ng-if="hyperlink.openNewTab()== false">
```

Comment:
Instead of nested ifs here, you can combine this into a single element and use more specific directives:
* `ng-class` will allow you to add a class to a element when something is true
* `ng-attr-target` will allow you to conditionally set the target.

For example, this could be:
```
<div >
    <a href="{{domainObject.getModel().url}}" 
       ng-attr-target="{{hyperlink.openNewTab() ? '_blank' : undefined}}"
       ng-class="{
           's-button': hyperlink.isButton()
       }"
       >
        {{domainObject.getModel().displayText}}
    </a>
</div>
```